JPS_Ontology/ontology/ontoschool/OntoSchool.csv

```diff
@@ -0,0 +1,38 @@
+Source,Type,Target,Relation,Domain,Range,Quantifier,Comment,Defined By,Label
+OntoSchool,TBox,https://www.theworldavatar.com/kg/ontoschool/,https://www.w3.org/2007/05/powder-s#hasIRI,,,,,,
+OntoSchool,TBox,1,http://www.w3.org/2002/07/owl#versionInfo,,,,,,
+OntoSchool,TBox,"OntoSchool is an ontology to represent schools and their information such as (contact information, address, CCAs etc).",http://www.w3.org/2000/01/rdf-schema#comment,,,,,,
+OntoSchool,TBox,http://ontology.eil.utoronto.ca/icontact.owl,http://www.w3.org/2002/07/owl#imports,,,,,,
+OntoSchool,TBox,https://raw.githubusercontent.com/cambridge-cares/TheWorldAvatar/main/JPS_Ontology/ontology/ontobuiltenv/OntoBuiltEnv.owl,http://www.w3.org/2002/07/owl#imports,,,,,,
+School,Class,https://w3id.org/bot#Building,IS-A,,,,An educational institution and building designed to provide learning spaces and learning environments for the teaching of students under the direction of teachers.,https://www.theworldavatar.com/kg/ontoschool/,School
```

Comment:
Interesting choice... but perhaps too restrictive? More generally, a school is an abstraction, and in many cases will comprise of several buildings. Some do not have any building at all (e.g. flexibly co-use other people's premises; or entirely online/remote teaching/learning). I believe if you remove "and building" from the explanation, then it would be fairly accurate. But subclass of bot building does not work in my view... Some kind of organisation/institution class (to which you could associate hasBuilding Building, if applicable!) would be more appropriate...